src/openforms/js/components/admin/design_token_values/index.js

```diff
@@ -1,9 +1,8 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
-import {IntlProvider} from 'react-intl';
+import {createRoot} from 'react-dom/client';
```

Comment:
It's one of the breaking changes in the React 18 upgrdade - the `createRoot` approach is essential to be able to make use of the new concurrent rendering/engine features - which we don't use currently, but new features like `startTransition` etc. will opt-in to concurrent features.

If I understand it correctly, then `ReactDOM.render` is also going to be removed at some point. This is a change on their side to also make the separation between server-side and client-side rendering more clear. We have no interest in SSR, only client-side stuff so that also makes it a bit more obvious.